media/distribution.html

```diff
@@ -19,14 +19,28 @@ <h2>Well-deployed technologies</h2>
-          <p>Media providers also need some mechanism to assess the user's perceived playback quality to alter the quality of content transmitted using adaptive streaming. The <a data-featureid="media-playback-quality">Media Playback Quality</a> specification, initially part of MSE, exposes metrics on the number of frames that were displayed or dropped.</p>
+          <p>On top of getting answers to <em>"Can"</em> questions, media providers also want answers to "<em>Should</em>" questions, such as "Should I sent HDR content if I have both HDR and SDR variants?". The Media Capabilities specification does not answer these questions, which rather fall within the realm of CSS extensions describing the capability of the display. Notably, <a data-featureid="color-gamut">CSS Media Queries Level 4</a> includes means to detect wide-gamut displays and adapt the rendering of the application to them. <a data-featureid="mediaqueries5">CSS Media Queries Level 5</a>, to be published after Level 4, will introduce a <code>video-</code> prefix to some features to detect differences between the video plane and the graphics plane on devices that render video separately from the rest of an HTML document, and that e.g. may support High Dynamic Range (HDR) for video and only Standard Dynamic Range (SDR) for other types of content.</p>
+          <p>Media providers also need some mechanism to assess the user's perceived playback quality in order to alter the quality of content transmitted using adaptive streaming. The <a data-featureid="media-playback-quality">Media Playback Quality</a> specification, initially part of MSE, exposes metrics on the number of frames that were displayed or dropped.</p>
+        </div>
+        <div data-feature="Adaptive streaming">
+          <p>A media stream may be composed of different programs that use different encodings, or may need to embed ads that have different characteristics. Applications cannot easily and smoothly merge these heterogeneous sub-streams back into one stream using the first version of Media Source Extensions (MSE), as it does not support this use case. Content providers need to work around this limitation, e.g. transcoding and re-packaging the ad content to be consistent with the program content prior to distribution where possible. The Media Working Group has started work on a <a data-featureid="mse-v2/codec-switching">codec switching feature for MSE</a> to address this issue.</p>
```

Comment:
I'm not sure this text quite captures the use case: "merge these heterogeneous sub-streams back into one stream", which to me suggests simultaneously combining together multiple streams. I suggest something like: "change from one media stream encoding to another during playback"

"work around this limitation" -> Another workaround is to use multiple video elements and script to show or hide them as the appropriate times